lib/Http/Request/Resolver/ResponseHeaderResolver.php

```diff
@@ -59,11 +59,7 @@ public function addResponseHeader(Request $request, string $key, array|string $v
         $responseHeaders = $this->getResponseHeaders($request);
 
         // manually add a @ResponseHeader config annotation to the list of headers
-        $responseHeaders[] = new ResponseHeader([
```

Comment:
Also here the line above
`// manually add a @ResponseHeader config annotation to the list of headers`